src/PowerShellEditorServices/Session/PowerShellContext.cs

```diff
@@ -943,22 +1186,56 @@ internal void BreakExecution()
 
         internal void ResumeDebugger(DebuggerResumeAction resumeAction)
         {
-            if (this.debuggerStoppedTask != null)
+            ResumeDebugger(resumeAction, shouldWaitForExit: true);
+        }
+
+        private void ResumeDebugger(DebuggerResumeAction resumeAction, bool shouldWaitForExit)
```

Comment:
No ResumeDebuggerAsync? :)

Reply:
Nah, iirc it's exclusively called from synchronous code.